zhmccli/_cmd_lpar.py

```diff
@@ -298,13 +301,19 @@ def cmd_lpar_deactivate(cmd_ctx, cpc_name, lpar_name):
     click.echo('Deactivation of LPAR %s is complete.' % lpar_name)
 
 
-def cmd_lpar_load(cmd_ctx, cpc_name, lpar_name, load_address):
+def cmd_lpar_load(cmd_ctx, cpc_name, lpar_name, load_address, options):
 
     client = zhmcclient.Client(cmd_ctx.session)
     lpar = find_lpar(client, cpc_name, lpar_name)
 
+    load_parameter = ""
+    options = original_options(options)
+    if options['load-parameter']:
+        load_parameter = options['load-parameter']
+
     try:
-        lpar.load(load_address, wait_for_completion=True)
+        lpar.load(load_address, load_parameter=load_parameter,
```

Comment:
Given that you use a generic approach for the options in the `cmd_lpar_load()` function, would it make sense to also do that when passing them to `lpar.load()` (unpacked as `**options`)?